Procfile.dev

```diff
@@ -1,2 +1,3 @@
+services: docker compose -f ./docker-compose.db.yml up
```

Comment:
cool !

To sync with PR [update developper doc](https://github.com/MTES-MCT/apilos/pull/818) which willl rename the docker-compose.db.yml file